src/form/elements/Select3.ts

```diff
@@ -147,6 +147,12 @@ export interface ISelect3Options<T extends Readonly<IdTextPair>> {
    * @default ' '
    */
   defaultTokenSeparator: string;
+
+  /**
+   * query delay
```

Comment:
Repeating the name of the variable in the comment is not really helpful. Could you please describe what this setting does and/or what the effect of a low/high value is?

```suggestion
   * Configures the time span how long to wait after a user has stopped typing before sending the AJAX request.
```